internal/resources/resource_networks.go

```diff
@@ -138,23 +109,39 @@ func Network() *schema.Resource {
 				Optional:    true,
 				Description: "If set to true, network will allow static override",
 			},
-			"config": {
+			"scope_id": {
```

Comment:
why is pool_id outside ? it should be under static_network config right ?

Reply:
yes updated